### ISSUE TITLE
overrideStyle seems to be missing (is not working)

### ISSUE THREAD

**Issue by fabiradi** [themre/roundy#8]
In `index.js`, the prop `overrideStyle` is not present on the Wrapper component. If I add the prop here, it ~~still seems to work only partly~~ works now (EDIT: I had two errors in my code, sorry):

- Setting a background works (after adding it inside `index.js`)
- ~~Changing the sliderHandle does not work (issue with styled-components?)~~
- ~~Changing the overall opacity does not work (same issue as above?)~~

```css
overrideStyle={`
    svg path { opacity: 1; } /* correcting fixed value of 0.7 */
    .sliderHandle:after {   /* Just testing */
        background: pink;
    }
`}
```

EDIT: Sorry, due to typos ans previously introduced createGlobalStyle (for svg opacity), I was responsible for two of the first mentioned issues. Now, I could actually create a pull request that completely fixes this issue ;-)

**Comment by fabiradi** [comment]
I created a pull request #9. So, this is not needed anymore. Sorry about the confusion ... :grimacing:

**Comment by william57m** [comment]
Same issue for me, I'm not able to use overrideStyle. See here https://codesandbox.io/s/roundy-examples-forked-o5u0x?file=/src/index.js

**Comment by themre** [comment]
oh, thanks for catching this, this was due to major refactor and using TS to rewrite everything. I published fix in version 3.0.3. Please give it a try!

**Comment by william57m** [comment]
Awesome it's working now, thanks for your reactivity!
